pkg/apb/watch_pod.go

```diff
@@ -91,3 +105,16 @@ func errorPullingImage(conds []apiv1.ContainerStatus) bool {
 
 	return false
 }
+
+func getExitCode(conds []apiv1.ContainerStatus) int32 {
```

Comment:
Just a suggestion. But as we know something went wrong because the pod failed, would it be clearer to have something like ```translateExitStatus(podName,  []apiv1.ContainerStatus)error ``` so that line 62 would become: 

```
return translateExitStatus(podName, podSatus.ContainerStatuses)
```
And the logic around exit codes would be contained to the single function. It would also avoid using the exit codes -120 and -127

Reply:
@maleck13 excellent suggestion, thanks.